core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/InternalPlugins.scala

```diff
@@ -3,9 +3,11 @@ package org.virtuslab.ideprobe.dependencies
 import org.virtuslab.ideprobe.BuildInfo
 
 object InternalPlugins {
-  val probePlugin: Plugin = bundle("ideprobe")
+  def probePlugin(version: String): Plugin = bundleCross("ideprobe", version)
 
-  val all = Seq(probePlugin)
+  def allForVersion(version: String) = Seq(probePlugin(version))
 
   def bundle(name: String): Plugin = Plugin.Bundled(s"$name-${BuildInfo.version}.zip")
```

Comment:
It would be a good method to use before, but I forgot to use it, now it is rather useless so you could delete it.
Also bundleCross has little use, so you might just want to inline that.